client/src/localization/languages/De.ts

```diff
@@ -325,8 +422,27 @@ export default {
   com_nav_search_placeholder: 'Nachrichten suchen',
   com_nav_setting_general: 'Allgemein',
   com_nav_setting_beta: 'Beta-Funktionen',
-  com_nav_setting_data: 'Daten-Kontrolle',
+  com_nav_setting_data: 'Datenkontrollen',
   com_nav_setting_account: 'Konto',
   com_nav_language: 'Sprache',
-  com_nav_lang_auto: 'Automatische Erkennung',
-};
+  com_nav_lang_auto: 'Auto detect',
+  com_nav_lang_english: 'English',
+  com_nav_lang_chinese: '中文',
+  com_nav_lang_german: 'Deutsch',
+  com_nav_lang_spanish: 'Español',
+  com_nav_lang_french: 'Français ',
+  com_nav_lang_italian: 'Italiano',
+  com_nav_lang_polish: 'Polski',
+  com_nav_lang_brazilian_portuguese: 'Português Brasileiro',
+  com_nav_lang_russian: 'Русский',
+  com_nav_lang_japanese: '日本語',
+  com_nav_lang_swedish: 'Svenska',
+  com_nav_lang_korean: '한국어',
+  com_nav_lang_vietnamese: 'Tiếng Việt',
+  com_nav_lang_traditionalchinese: '繁體中文',
+  com_nav_lang_arabic: 'العربية',
+  com_nav_lang_turkish: 'Türkçe',
+  com_nav_lang_dutch: 'Nederlands',
+  com_nav_lang_indonesia: 'Indonesia',
+  com_nav_lang_hebrew: 'עברית',
```

Comment:
Would you mind removing these translations?

Reply:
i removed it from my fork, sry

Reply:
no problem, thanks